### DISCUSSION TITLE
ReportMissingModuleSource

### DISCUSSION THREAD

**Discussion by raracalma** [microsoft/pylance-release#3064] [General]
I have a problem: my program is unable to find the library "matplotlib".  When running the code, the problem appears: 'Import "matplotlib.pyplot" could not be resolved from source Pylance(reportMissingModuleSource).
how do I solve this problem (which I know is quite amateur, but I'm learning now)?

![Captura de tela 2022-07-18 174659](https://user-images.githubusercontent.com/109553231/179614474-e3f8ea6a-afcf-41fd-849d-b3772bb7649e.png)

**Comment by debonte** [comment]
Have you installed matplotlib in your virtual environment? This information may help: https://code.visualstudio.com/docs/python/environments
